posts/2024-07-30-24.0.0.8-beta.adoc

```diff
@@ -0,0 +1,816 @@
+
+For more information on using a beta release, refer to the link:{url-prefix}docs/latest/installing-open-liberty-betas.html[Installing Open Liberty beta releases] documentation.
+
+[#feedback]
+== We welcome your feedback
+
+Let us know what you think on link:https://groups.io/g/openliberty[our mailing list]. If you hit a problem, link:https://stackoverflow.com/questions/tagged/open-liberty[post a question on StackOverflow]. If you hit a bug, link:https://github.com/OpenLiberty/open-liberty/issues[please raise an issue].
+
+[#platform-ref]
+== Platforms and related versionless features reference list
+
+The following platform features are available in this beta release. You can specify up to 2 platform elements, one for MicroProfile, another for either Jakarta EE or Java EE.
+
+=== MicroProfile
+
+Platforms:
+
+* `microProfile-1.2`
+* `microProfile-1.3`
+* `microProfile-1.4`
+* `microProfile-2.0`
+* `microProfile-2.1`
+* `microProfile-2.2`
+* `microProfile-3.0`
+* `microProfile-3.2`
+* `microProfile-3.3`
+* `microProfile-4.0`
+* `microProfile-4.1`
+* `microProfile-5.0`
+* `microProfile-6.0`
+* `microProfile-6.1`
+
+MicroProfile versionless features:
+
+* `mpConfig`
+* `mpFaultTolerance`
+* `mpHealth`
+* `mpJwt`
+* `mpMetrics`
+* `mpOpenAPI`
+* `mpOpenTracing`
+* `mpRestClient`
+* `mpTelemetry`
+
+
+==== Jakarta EE
+
+Platforms:
+
+* `jakartaee-9.1`
+* `jakartaee-10.0`
+
+Jakarta EE versionless features:
+
+* `appAuthentication`
+* `appAuthorization`
+* `appClientSupport`
+* `appSecurity`
+* `batch`
+* `beanValidation`
+* `cdi`
+* `concurrent`
+* `connectors`
+* `connectorsInboundSecurity`
+* `data`
+* `enterpriseBeans`
+* `enterpriseBeansHome`
+* `enterpriseBeansLite`
+* `expressionLanguage`
+* `enterpriseBeansPersistentTimer`
+* `enterpriseBeansRemote`
+* `faces`
+* `j2eeManagement`
+* `jdbc`
+* `jsonb`
+* `jsonp`
+* `mail`
+* `managedBeans`
+* `mdb`
+* `messaging`
+* `messagingClient`
+* `messagingSecurity`
+* `messagingServer`
+* `pages`
+* `persistence`
+* `restfulWS`
+* `restfulWSClient`
+* `servlet`
+* `websocket`
+* `xmlBinding`
+* `xmlWS`
+
+==== Java EE
+
+Platforms:
+
+* `javaee-6.0`
+* `javaee-7.0`
+* `javaee-8.0`
+
+Java EE versionless features:
+
+* `appClientSupport`
+* `appSecurity`
+* `batch`
+* `beanValidation`
+* `cdi`
+* `concurrent`
+* `jca`
+* `jcainboundsecurity`
+* `data`
+* `ejb`
+* `ejbhome`
+* `ejblite`
+* `el`
+* `ejbpersistenttimer`
+* `ejbremote`
+* `jacc`
+* `jaspic`
+* `javaMail`
+* `jaxb`
+* `jaxws`
+* `jaxrs`
+* `jaxrsclient`
+* `jpa`
+* `jsf`
+* `jsp`
+* `j2eeManagement`
+* `jdbc`
+* `jms`
+* `jsonb`
+* `jsonp`
+* `managedBeans`
+* `mdb`
+* `servlet`
+* `websocket`
+* `wasjmsclient`
+* `wasjmssecurity`
+* `wasjmsserver`

```

Comment:
```suggestion
* `ejb`
* `ejbHome`
* `ejbLite`
* `ejbPersistentTimer`
* `ejbRemote`
* `el`
* `j2eeManagement`
* `jacc`
* `jaspic`
* `javaMail`
* `jaxb`
* `jaxrs`
* `jaxrsClient`
* `jaxws`
* `jca`
* `jcaInboundSecurity`
* `jdbc`
* `jms`
* `jpa`
* `jsf`
* `jsonb`
* `jsonp`
* `jsp`
* `managedBeans`
* `mdb`
* `servlet`
* `wasJmsClient`
* `wasJmsSecurity`
* `wasJmsServer`
* `websocket`
```

In reality these can be combined and say Java EE and Jakarta EE platforms and then in the bulleted points do:

* `ejb` or `enterpriseBeans`

We support both with Java or Jakarta EE and don't require you to use Java EE feature names for Java EE platforms or vice versa.  You can mix and match.  @cbridgha and @rsherget what do you think?

Reply:
There will still be both versions of the feature names in the GA.  You can use the javaee or jakartaee names for any of the javaee / jakartaee platform names.